src/test/groovy/page/sessionPage/SessionPage.groovy

```diff
@@ -0,0 +1,150 @@
+package page.sessionPage
+
+import geb.Page
+
+
+class SessionPage extends Page {
+
+    static url = "Session/AddSession"
+
+    static at = {
+        title == "- ExamPlanner"
+        sessionForm.displayed
+    }
+
+    static content = {
+
```

Comment:
W gebie są natywne moduły do obsługi formularzy: http://www.gebish.org/manual/current/#form-control-modules
Chociaż w sumie nie musisz tego zmieniać, skoro już mamy to napisane :)